scripts/build.clj

```diff
@@ -9,7 +9,7 @@
     :output-to "out/tests.js"
     :output-dir "out"
     :target :nodejs
-    :optimizations :advanced
+    :optimizations :none
```

Comment:
As we discussed today, this is ok for temporal workaround, but advanced optimizations should be activated for tests, because this is the unique way to test that users can use this library with advanced compilations.